### PR TITLE
search

### PR DESCRIPTION
**Pull request by haihv96**

### CODE REVIEW

**Comment by haihv96** [comment]
ready

File: app/controllers/searchs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class SearchsController < ApplicationController
+  def index
+    return unless params[:search][:type] == "users"
+    keyword = "%#{params[:search][:keyword]}%"
+    @users = User.where("full_name LIKE BINARY :keyword OR email LIKE :keyword
+       OR phone LIKE :keyword", keyword: keyword)
```
Comment (huongnguyenmta) [inline app/controllers/searchs_controller.rb]: indent

File: app/controllers/searchs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class SearchsController < ApplicationController
+  def index
+    return unless params[:search][:type] == "users"
+    keyword = "%#{params[:search][:keyword]}%"
+    @users = User.where("full_name LIKE BINARY :keyword OR email LIKE :keyword
```
Comment (huongnguyenmta) [inline app/controllers/searchs_controller.rb]: Viết scope nhé

File: app/views/shared/_users_json.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<% @users.each do |user| %>
+  <div class="item-user-search col-md-12">
+    <%= link_to user, class: "black" do %>
+      <div class="pull-left">
+        <%= avatar_tag user, width: 55 %>
```
Comment (huongnguyenmta) [inline app/views/shared/_users_json.html.erb]: move `55` into setting

**Comment by huongnguyenmta** [comment]
commented

**Comment by haihv96** [comment]
ready

**Comment by huongnguyenmta** [comment]
merged
(dance2)

**Pull request merged**